oci/common.go

```diff
@@ -136,56 +136,19 @@ func (o *OrasRemote) setRepository(ref registry.Reference) error {
 		ref.Registry = "ghcr.io"
 		ref.Repository = "defenseunicorns/packages/" + ref.Repository
 	}
-	client, err := o.createAuthClient(ref)
+	storeOpts := credentials.StoreOptions{}
+	credStore, err := credentials.NewStoreFromDocker(storeOpts)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to get credentials: %w", err)
+	}
+	client := &auth.Client{
+		Client:     retry.DefaultClient,
+		Cache:      auth.NewCache(),
+		Credential: credentials.Credential(credStore),
 	}
 
 	o.repo.Reference = ref
 	o.repo.Client = client
 
 	return nil
 }
-
-// createAuthClient returns an auth client for the given reference.
-//
-// The credentials are pulled using Docker's default credential store.
-//
-// TODO: instead of using Docker's cred store, should use the new one from ORAS to remove that dep
-func (o *OrasRemote) createAuthClient(ref registry.Reference) (*auth.Client, error) {
-
-	client := o.repo.Client.(*auth.Client)
-	o.log.Debug(fmt.Sprintf("Loading docker config file from default config location: %s for %s", config.Dir(), ref))
-	cfg, err := config.Load(config.Dir())
-	if err != nil {
-		return nil, err
-	}
-	if !cfg.ContainsAuth() {
-		o.log.Debug("no docker config file found")
-		return client, nil
-	}
```

Comment:
do we want to keep this debug message using `func (ds *DynamicStore) IsAuthConfigured() bool {`?

Reply:
Good catch, I would say yes, I'm going to add that to the Crane->ORAS Zarf PR as well actually